src/models/login/index.ts

```diff
@@ -0,0 +1,32 @@
+import mongoose, { Schema } from "mongoose";
+import { ILogin } from "./type";
+
+const schema = new mongoose.Schema(
+  {
+    userName: {
+      type: String,
+      required: true,
+      unique: true,
+      trim: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+      trim: true,
+    },
+    accessToken: {
+      type: String,
+      required: true,
+    },
+    _id: {
```

Comment:
I have a question here. Is _id the id of session object ? If so then _id will get auto generated and we dont  need to define it. If _id means value of some other objects (.  lets say user Obj ) then rename the _id to `userId` to avoid confusion.

Reply:
Yes added userId field which will have _id of user model and _id for login model will get auto generated.